README.md

```diff
@@ -16,6 +16,12 @@ The **Azure IoT Edge Dev Tool** enables you to do all of the following with simp
 
     `iotedgedev project --create edgeproject1`
 
+1. **Azure**: Setup your new IoT Edge project configuration. This will guide you to select or create the needed Azure resources and will populate the required Environment Variables.
+
+    `iotedgedev azure --setup --subscription THE_SUBSCRIPTION_ID --resource-group-name THE_RG_NAME --iothub-name THE_IOT_NAME --edge-device-id THE_EDGE_DEVICE_NAME`
```

Comment:
Let's flip this.  Show --setup without any parameters.  And then show how you can also call with all the parameters inline later in the doc.  I want the intro section to be very clean.

Change to: Creates or selects your Azure IoT Hub and Edge Device and updates your Environment Variables.